packages/dd-trace/test/appsec/remote_config/index.spec.js

```diff
@@ -340,6 +340,7 @@ describe('Remote Config index', () => {
           .to.have.been.calledWithExactly(RemoteConfigCapabilities.ASM_RASP_SQLI, true)
         expect(rc.updateCapabilities)
           .to.have.been.calledWithExactly(RemoteConfigCapabilities.ASM_RASP_LFI, true)
+          .to.have.been.calledWithExactly(RemoteConfigCapabilities.ASM_RASP_SHI, true)
```

Comment:
Maintain consistency by adding the `expect` and not concatenating it to the previous one.
```suggestion
         expect(rc.updateCapabilities)
          .to.have.been.calledWithExactly(RemoteConfigCapabilities.ASM_RASP_SHI, true)
```

Reply:
oh, it was not intentional,  I didn't know even that it could be joined 🤦